magento/magento2
# Added: SKU with configurable, group, bundle product Title

### Description (*)

Changed Magento/Paypal/Model/Cart.php file to add child product SKUs with the Parent product title to identify the product title with the child SKU. 
Check this screenshot for more information: https://prnt.sc/0VgjbwOg-omA

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#36808

### Manual testing scenarios (*)

1. Enable PayPal Express Checkout [sandbox or live, the result is the same so just do sandbox for this test]
2. Enable "Transfer Cart Line Items"
3. Enable "PayPal Guest Checkout"
4. Submit the Order (by credit card using a test number provided [HERE](https://developer.paypal.com/api/rest/sandbox/card-testing)). It is the same end result using a PayPal personal account but for testing purposes just use a test credit card for easier testing.
5. Check PayPal transactions in the PayPal sandbox account (not Magento)

### Questions or comments


### Contribution checklist (*)
